CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Release Notes
 
+## Nerves v0.9.0
+
+* Update Notes
+
+Starting in Nerves v0.9.0, artifacts will no longer be fetched during
+`mix compile`. Artifact archives are intended to be fetched following `mix deps.get`.
+To handle this, you will need to update your installed version of `nerves_bootstrap` by
+calling `mix nerves.local`. After updating nerves_bootstrap, you should update 
+your mix.exs file to add the new required mix aliases found there. A helper 
```

Comment:
I assume we should tick mix.exs here too

Reply:
yeah basically everywhere or nowhere.